src/mesh/webgl/MeshRenderer.js

```diff
@@ -98,7 +99,17 @@ export default class MeshRenderer extends core.ObjectRenderer
             glData.indexBuffer.upload(mesh.indices);
         }
 
-        glData.vertexBuffer.upload(mesh.vertices);
+        // For backwards compatibility
+        if (mesh.autoVertexUpdate)
```

Comment:
This increment has to be moved to refresh(): https://github.com/pixijs/pixi.js/blob/dev/src/mesh/Mesh.js#L211

Reply:
Isn't it the same as the one for `indicies` and `uvs`? I see that the refresh function is a noop.

Reply:
refresh function is called from renderWebGL. I know that your code works, i just wantto move that particular part to Mesh itself, because its his updateID.

Reply:
At this moment, refresh takes care of texture changes, and forces recaltulation of uvTransform if needed. ITs not fully no-op and its the only function there that does something with updates.

Reply:
Thoughts @Cristy94?

Reply:
If that refresh function is called each tick it should be fine. I didn't have the time to go back to it and make the changes yes.